### PR TITLE
spelling & grammar changes

### PR DESCRIPTION
**Pull request by markvader**
Just came to fix one spelling, but then I got distracted a fixed a few more

### CODE REVIEW

File: hassio-google-drive-backup/backup/static/layouts/partials/error-messages.jinja2 [hunk 1]
```diff
@@ -653,8 +653,8 @@
 {% call macros.errorMessage("Google drive Returned an Unexpected Response", "google_unexpected", "error", google_unexpected_actions) %}
   <p>
     The addon was unable to communicate with Google Drive because it gave back an unreadable response.  
-    Withotu beign able to communicate with google drive, the addon will nto be able to back up backups to the cloud.
-    This usually resolves itself after the addon retrys, but if you see this error stick around for a while you can try:
+    Without being able to communicate with Google Drive, the addon will not be able to transfer backups to the cloud.
```
Comment (sabeechen) [inline hassio-google-drive-backup/backup/static/layouts/partials/error-messages.jinja2:656]: Good call wrt "transfer".  I've struggled to change all the relevan6t language in the add-on after they renamed "snapshots" to "backups"

File: hassio-google-drive-backup/backup/static/working.jinja2 [hunk 2]
```diff
@@ -203,23 +203,23 @@
                If you are affected this means that in October 2022 the add-on will be unable to authorize with Google Drive and your backups will not get uploaded.  
             </p>
             <p>
-              While annoying, this change is also somewhat expected.  Using using Google Cloud credentials for a local installation like this is not something Google 'intended' for us to do, it isn't an officially supported way of using their services. We can still make due though.  
+              While annoying, this change is also somewhat expected.  Using Google Cloud credentials for a local installation like this is not something Google 'intended' for us to do, it isn't an officially supported way of using their services. We can still make do though.
```
Comment (sabeechen) [inline hassio-google-drive-backup/backup/static/working.jinja2:206]: I had to look this one up.  Turns out I've been spelling "make do" wrong for over 30 years.

**Comment by sabeechen** [comment]
Thanks for putting in the work to do this.  I can only speculate as to what could possibly motivate you to scrutinize every body of text in this web-ui.

It may not surprise you to know that [typo corrections are by far the most common contribution ](https://github.com/sabeechen/hassio-google-drive-backup/pulls?q=typo)made to this project.  Seems like no matter how hard I re-read over things, I can just never catch them all.

**Comment by markvader** [comment]
> Thanks for putting in the work to do this. I can only speculate as to what could possibly motivate you to scrutinize every body of text in this web-ui.
> 
> It may not surprise you to know that [typo corrections are by far the most common contribution ](https://github.com/sabeechen/hassio-google-drive-backup/pulls?q=typo)made to this project. Seems like no matter how hard I re-read over things, I can just never catch them all.

Glad I could help. 
I think we are all the same with our own writing, complete typo blindness because we know what it should say.

Thank you for your all your work on the project. Such an vital add-on that you have provided

**Pull request merged**